src/Kentico.Xperience.AiraUnified/Admin/UIPages/AiraConfigurationEditPage.cs

```diff
@@ -78,19 +78,31 @@ protected override async Task<ICommandResponse> ProcessFormData(AiraUnifiedConfi
 
     private static bool IsValidSubpath(string path)
```

Comment:
Also I am just thinking ... what about creating an extension method to string type that would handle this, I think it would make the code somehow nicer